test/functional-portable/README.md

```diff
@@ -0,0 +1,3 @@
+* [Writing functional tests](../../docs/contributing/contributing-code/contributing-code-tests/writing-functional-tests.md)
+
```

Comment:
Is there much value to this file? Open to opinions here (deleting this is an option). 

Here's what it looks like:

<img width="359" alt="image" src="https://github.com/radius-project/radius/assets/1430011/2dcce8c7-3628-44d8-8b91-fcd1fee2dd81">


My reaction is that this file doesn't provide a lot of context. 

- Contributors who are familiar with the repo will know how to find these documents, or already know their content.
- Contributors who are new to Radius won't get much value of this because the context it provides is limited.

Reply:
I agree.... I will remove it. Doesn't make too much sense to keep it here.